js/constants/appConfig.js

```diff
@@ -33,7 +33,8 @@ module.exports = {
     COOKIEBLOCK: 'cookieblock', // block 3p cookies and referer
     COOKIEBLOCK_ALL: 'cookieblockAll', // block all cookies and referer
     SITEHACK: 'siteHacks',
-    WEBTORRENT: 'webtorrent'
+    WEBTORRENT: 'webtorrent',
+    IPFS: 'ipfs'
```

Comment:
This might no longer be necessary, as now the extension is loaded as a background process.